docs/wiki/frameworkx/fortify-framework.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: wiki
+title: Fortify Framework
+description: Explains how to set-up the Fortify component.
+group: framework
+order: 6
+parent: wiki
+mod: acex
+version:
+  major: 3
+  minor: 3
+  patch: 0
+---
+
+## 1. Overview
+
+The Fortify framework allows server admins or mission makers to give players the ability to place fortifications through self-interaction, using the `Fortify Tool`.
+
+
+### 1.1 Chat commands
+
+If the Fortify module is present in the mission server admins can use chat commands to set-up or change the different parameters. Useful to give players additional resources based on progress on the mission for instance.
+
+- `#fortify on` turns fortify mode on
```

Comment:
It's actually `#ace-fortify ...`, was changed later in the PR.

Reply:
On all ?